ios/bottom-tabs/RNSBottomTabsScreenComponentViewManager.mm

```diff
@@ -23,7 +23,7 @@ - (UIView *)view
 RCT_EXPORT_VIEW_PROPERTY(title, NSString);
 RCT_EXPORT_VIEW_PROPERTY(badgeValue, NSString);
 RCT_EXPORT_VIEW_PROPERTY(tabBarBackgroundColor, UIColor);
-RCT_EXPORT_VIEW_PROPERTY(tabBarBlurEffect, UIBlurEffect); // needs custom conversion, cause it has `none`
+RCT_REMAP_VIEW_PROPERTY(tabBarBlurEffect, tabBarBlurEffectFromRNSBlurEffectStyle, RNSBlurEffectStyle);
```

Comment:
smart

let's add some comment with clarification, so that we still know why it is remapped this way in few weeks.